exam_marker/Dockerfile

```diff
@@ -1,5 +1,28 @@
-FROM golang:1.17
-MAINTAINER Christian Heusel <christian@heusel.eu>
+FROM golang:1.17.6-bullseye
```

Comment:
Is there a reason to go for a fixed go version? I would rather use the `1.17-bullseye` tag!